EcpBtBot/DialogFlow/Response.py

```diff
@@ -19,7 +19,7 @@ def status(self):
 class Result:
     def __init__(self,result):
         self.result= result
-        self.action=result['action']
+        #self.action=result['action']
         self.source=result['source']
```

Comment:
spaces around `=`.

It seems that you are using visual studio, which is fine. Probably there is there what is called a `linter` which is a program that checks the syntax and sometimes more in your code and highlights what is wrong. If you use one, it will help you follow such guidelines